src/pages/detail.vue

```diff
@@ -65,26 +68,36 @@ export default {
     ...mapGetters([
       'user',
     ]),
-    ago() {
-      return ago.format(new Date(this.enquiry.createdAt))
-    },
-    enquiryOwner() {
-      return this.userLookup(this.enquiry.owner)
-    },
+
+    // enquiryOwner() {
+    //   return this.userLookup(this.enquiry.owner)
+    // },
   },
 
   async mounted() {
-    console.log('getting responses')
     this.$f7.dialog.preloader('Loading enquiry...')
-    this.enquiry = await this.getEnquiry(this.enquiryId)
+
+    // pull a fresh copy of enquiry and responses to get warrior details
+    // we go around the store here because this data won't be used elsewhere
+    const [enquiry, responses] = await Promise.all([
```

Comment:
If we are going to go a round the store here, I'd much rather see functions called in the mount.... Just makes things a bit more clear and cleaner to read IMO.